docs/pages/installation.mdx

```diff
@@ -439,15 +439,15 @@ either:
   `(=teleport.version=)`.
 
 <Tabs>
-<TabItem label="Teleport Community Edition" scope={["oss"]}>
+<TabItem label="Teleport Enterprise Cloud" scope={["team", "cloud"]}>
```

Comment:
We're now using "Teleport Enterprise" for both cloud-hosted and self-hosted deployments, so I would say "Managed Teleport Enterprise" here.